packages/react/src/components/ContentBlockMedia/__stories__/ContentBlockMedia.stories.js

```diff
@@ -157,7 +157,7 @@ WithAsideElements.story = {
         const items = [item, item];
 
         const linkListProps = {
-          heading: text('link list heading:', 'Tutorials', groupId),
+          heading: text('link list (heading)', 'Tutorials', groupId),
```

Comment:
@andysherman2121 thanks to add it. Just another minor comment... For the `link list`, is CamelCase good ? Like this: `Link List`.

Reply:
@diegomarcuz hmm yeah maybe it should be `Link List (heading)`

Reply:
Awesome, thanks @andysherman2121